examples/pytorch_imagenet_resnet50.py

```diff
@@ -286,7 +170,132 @@ def avg(self):
         return self.sum / self.n
 
 
-for epoch in range(resume_from_epoch, args.epochs):
-    train(epoch)
-    validate(epoch)
-    save_checkpoint(epoch)
+if __name__ == '__main__':
```

Comment:
What was the reason for moving the code in this file under __main__, but not the other examples?